src/components/MultiHopTrade/hooks/useTradeExecution/useTradeExecution.ts

```diff
@@ -153,6 +153,10 @@ export const useTradeExecution = ({
           quoteId: tradeQuote.id,
           txHash: sellTxHashRef.current,
           chainId,
+          stepIndex: activeStepOrDefault,
+          quoteSellAssetAccountId: sellAssetAccountId,
+          quoteBuyAssetAccountId: buyAssetAccountId,
+          getState: store.getState,
```

Comment:
@woodenfurniture FYI this is effectively becoming `checkStepStatus` - we might want to revamp the vernacular here